src/Microsoft.DotNet.Build.Tasks/PackageFiles/buildagainstpackages.targets

```diff
@@ -177,6 +177,11 @@
     <PropertyGroup>
         <PackageBuildNumberOverride Condition="'$(PackageBuildNumberOverride)' == '' and '$(_DerivedPackageNumber)' != ''">$(_DerivedPackageNumber)</PackageBuildNumberOverride>
     </PropertyGroup>
+    <!-- we don't yet have runtime packages for these RIDs so exclude them for now -->
+    <ItemGroup>
+      <ExcludedRuntimes Include="fedora.23-x64" />
```

Comment:
Agreed.  I will see where we can define these in CoreFX (probably the sync project that imports this target).